src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSIApplication.java

```diff
@@ -638,4 +648,141 @@ public void verifyThatTheGuardarButtonIsNotAvailableIfTheNombreDeAlmacenFieldIsE
     public void hePageLoadedMustContainAsTitle(String title) throws Throwable {
         Assert.assertEquals(storageReport.getTitle(), title, String.format(ErrorMessage.ERROR_MESSAGE_NAME_ELEMENT_PRESENT, title));
     }
+
+    @And("^Click 'Login' button on Page Header$")
+    public void clickLoginButtonOnPageHeader() throws Throwable {
+        if (navigationBar == null) {
+            login.clickLoginButton();
+        }
+    }
+
+    @And("^Enter Username$")
+    public void enterUsername() throws Throwable {
+        if (navigationBar == null) {
+            login.enterUsername();
+        }
+    }
+
+    @And("^Enter Password$")
+    public void enterPassword() throws Throwable {
+        if (navigationBar == null) {
+            login.enterPassword();
+        }
+    }
+
+    @And("^Click 'Login' button to accept credentials$")
+    public void clickLoginButtonToAcceptCredentials() throws Throwable {
+        if (navigationBar == null) {
+            navigationBar = login.clickLoginAcceptButton();
+        }
+    }
```

Comment:
Let's reuse this steps  for login to the application : 
 @Given("^'SSI Application' page is loaded$")
    public void ssiApplicationPageIsLoaded() throws Throwable {
        login = LoadPage.loginPage();
    }

    @And("^set Admin credentials on 'Login' page$")
    public void setCredentialsOnLoginPage() throws Throwable {
        if (navigationBar == null) {
            navigationBar = login.setCredentials();
        }
    }